pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -77,8 +80,82 @@ func (pl *CSILimits) Name() string {
 func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
 		{Event: framework.ClusterEvent{Resource: framework.CSINode, ActionType: framework.Add}},
-		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}},
+		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}, QueueingHintFn: pl.isSchedulableAfterPodDeleted},
+	}
+}
+
+func (pl *CSILimits) isSchedulableAfterPodDeleted(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	deletedPod, _, err := util.As[*v1.Pod](oldObj, newObj)
+	if err != nil {
+		logger.V(5).Info("Unexpected objects in isSchedulableAfterPodDeleted", "oldObj", oldObj, "err", err)
```

Comment:
logger.Error in all error cases.

Reply:
Wait No, we don't need such error logs actually because the scheduling queue emits error logs when a plugin returned errors in QHint.

Reply:
Okay. Should I remove all logs related to the error?

Reply:
Yes

Reply:
https://github.com/kubernetes/kubernetes/pull/121508/commits/243a05275db60b83724cc27600983148bd5cbcd6